### PR TITLE
VIH-9236 - Correct Secrets

### PR DESCRIPTION
**Pull request by chris-pateman**
### JIRA link (if applicable) ###
VIH-9236

### Change description ###
Correct Secrets

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: charts/vh-admin-web/values.yaml [hunk 1]
```diff
@@ -28,27 +28,52 @@ java:
...
-    FEATURETOGGLE__SDKKey: "PLACEHOLDER"
     ISLIVE: "False"
     Logging__LogLevel__Default: debug
     Logging__LogLevel__System: debug
     Logging__LogLevel__Microsoft: debug
     TESTUSERSECRETS__TESTUSERNAMESTEM: "@hearings.reform.hmcts.net"
     TESTUSERSECRETS__TESTUSERPASSWORD: "PLACEHOLDER"
-    VHSERVICES__BOOKINGSAPIRESOURCEID: https://vh-bookings-api.{{ .Values.global.environment }}.platform.hmcts.net
-    VHSERVICES__BOOKINGSAPIURL: https://vh-bookings-api.{{ .Values.global.environment }}.platform.hmcts.net/
     VHSERVICES__LAUNCHDARKLYCLIENTID: "PLACEHOLDER"
```
Comment (v1ferrarij) [inline charts/vh-admin-web/values.yaml:73]: are these not in the KVs? --referring to "PLACEHOLDER"

**Review by v1ferrarij — CHANGES_REQUESTED** [review]

File: charts/vh-admin-web/values.yaml (hunk 1 above)
Comment (chris-pateman) [reply charts/vh-admin-web/values.yaml:74]: latest change did not push up :)

File: AdminWebsite/AdminWebsite/Program.cs [hunk 2]
```diff
@@ -18,12 +19,23 @@ private static IHostBuilder CreateWebHostBuilder(string[] args)
         {
             const string vhInfraCore = "/mnt/secrets/vh-infra-core";
```
Comment (v1ferrarij) [inline AdminWebsite/AdminWebsite/Program.cs:20]: Do we need these two lines if we are looping through them all now?

**Review by v1ferrarij — COMMENTED** [review]
we dont need the const strings now

File: AdminWebsite/AdminWebsite/Program.cs (hunk 2 above)
Comment (chris-pateman) [reply AdminWebsite/AdminWebsite/Program.cs]: good point no.

**Review by v1ferrarij — APPROVED** [review]

**Pull request merged**